astrolabe/atlas_runner.py

```diff
@@ -570,8 +559,9 @@ def clean_old_projects(self, org_id):
                         else:
                             raise
                 else:
+                    LOGGER.info("Skipping deleting project %r, id: %r", project.name, project.id)
                     LOGGER.info(
-                        "Skipping deleting project %r, id: %r", project.name, project.id
+                        f"{project_timestamp=}, {current_timestamp=}, {self.project_expiration_threshold_seconds=}"  # noqa: G004
```

Comment:
@blink1073, @matthewdale  this actually snaked in, I wonder if we should still keep it as a valuable debugging datapoint

Reply:
Yes I think so